number_box: 1
distance_hand: 0.02
discrete_side: 10
scale_factor: 1000